test/nerves_time/rtc/abracon/id_test.exs

```diff
@@ -2,8 +2,10 @@ defmodule NervesTime.RTC.Abracon.IDTest do
   use ExUnit.Case
   alias NervesTime.RTC.Abracon.ID
 
+@ab_08x5 <<0x08, 0x05, 0x13>>
+
   test "decodes ID" do
-    assert ID.decode(<<24, 5, 19, 49, 70, 89, 48>>) ==
-             {:ok, %{id: :ab_rtcmc_32768khz_ibo5_s3, lot: 49, part_id: 18009, wafer: 12}}
+    assert ID.decode(<<0x18, 0x05, 0x13, 49, 70, 89, 48>>) ==
+             {:ok, %{id: @ab_08x5, lot: 49, part_id: 18009, wafer: 12}}
```

Comment:
ℹ️ (optional)

If this were added as an additional test to support the new AB08X5 part in addition to the prior IBO5-S3, it might be possible to upstream this PR.